netbox/dcim/models/__init__.py

```diff
@@ -435,9 +435,20 @@ def _draw_empty(drawing, rack, start, end, text, id_, face_id, class_, reservati
 
     def _draw_elevations(self, elevation, reserved_units, face, unit_width, unit_height):
 
-        drawing = self._setup_drawing(unit_width, unit_height * self.u_height)
+        # Setup vars for the rack unit
+        position_width = 30
+
+        drawing = self._setup_drawing(unit_width + position_width, unit_height * self.u_height)
 
         unit_cursor = 0
+        for ru in range(0, self.u_height):
+            start_y = ru * unit_height
+            position_coordinates = (15, start_y + unit_height / 2 + 2)
```

Comment:
Where did the `15` come from?

Reply:
Half of 30, since we are using a variable, changed it to that